nix-tools/static/packaging.nix

```diff
@@ -0,0 +1,115 @@
+super: self: 
```

Comment:
this was pretty much copy-pasted from `haskell-nix-example`, The only change was to the `asZip` function which now takes an attrset instead of separate arguments.